lib/order.rb

```diff
@@ -0,0 +1,105 @@
+# I think these requires are only required for my personal testing????
+require 'csv'
+require_relative 'customer.rb'
+
+class Order
+  attr_reader :id, :customer
+  attr_accessor :products, :fulfillment_status
+  def initialize(id, product_hash, customer, fulfillment_status = :pending)
+    @id = id
+    @products = product_hash
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+
+    valid_statuses = [:pending, :paid, :processing, :shipped, :complete]
+    if !valid_statuses.include? @fulfillment_status
```

Comment:
For checks like this you can use `unless CONDITION` instead of `if !CONDITION` which reads a little clearer since it's easy to overlook the `!`

```suggestion
    unless valid_statuses.include? @fulfillment_status
```